src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -257,9 +262,11 @@ class ChatSwitcherView extends React.Component {
         // the user that was just selected and everyone in the list
         if (this.state.usersToStartGroupReportWith.length > 0) {
             const userLogins = _.map(this.state.usersToStartGroupReportWith, option => option.login);
-            fetchOrCreateChatReport([this.props.session.email, ...userLogins, selectedOption.login]);
+            fetchOrCreateChatReport([this.props.session.email, ...userLogins, selectedOption.login])
+                .then(this.props.onReportSelected);
         } else {
-            fetchOrCreateChatReport([this.props.session.email, selectedOption.login]);
+            fetchOrCreateChatReport([this.props.session.email, selectedOption.login])
+                .then(this.props.onReportSelected);
```

Comment:
Just FYI this entire file will be removed very soon and this change makes it difficult to do.